portal/src/main/webapp/js/src/civic/civicservice.js

```diff
@@ -0,0 +1,166 @@
+
+function CivicService() {
+    
+    // All results are stored here
+    var civicGenes = {};
+    
+    function getCivicGenesBatch(ids) {
+        return $.ajax({
+            type: 'GET',
+            url: 'api-legacy/proxy/civic/genes/' + ids,
```

Comment:
Since civic is on https, we don't have to use a proxy. @kretep is it possible to connect directly?